recipes/python-avro/meta.yaml

```diff
@@ -4,7 +4,7 @@ package:
   name: python-avro
   version: {{ version }}
 
-source:
+source:  # [py27 or py3k]
```

Comment:
Thus if it is not Python 2.7 or Python 3.x, this section doesn't appear. In those cases, this section is empty anyways. This will workaround the `conda-build` issue.